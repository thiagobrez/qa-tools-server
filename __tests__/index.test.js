const request = require('supertest');
const app = require('../app');

it('reads all users', async () => {
  const res = await request(app).get('/');

  expect(res.status).toBe(200);
  expect(res.text).toMatchSnapshot();
});