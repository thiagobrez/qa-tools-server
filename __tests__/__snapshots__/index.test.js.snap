// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reads all users 1`] = `
"<html>

<head>
  <title>Express</title>
  <link rel=\\"stylesheet\\" href=\\"/stylesheets/style.css\\">
</head>

<body>
  <h1>Express</h1>
  <p>Welcome to Express</p>
</body>

</html>
"
`;
